build.gradle

```diff
@@ -973,10 +973,28 @@ project(':iceberg-open-api') {
     testFixturesImplementation project(':iceberg-api')
     testFixturesImplementation project(':iceberg-core')
     testFixturesImplementation project(path: ':iceberg-core', configuration: 'testArtifacts')
-    testFixturesImplementation project(':iceberg-core').sourceSets.test.runtimeClasspath
     testFixturesImplementation project(':iceberg-aws')
     testFixturesImplementation project(':iceberg-gcp')
     testFixturesImplementation project(':iceberg-azure')
+    testFixturesImplementation(libs.hadoop3.common) {
+      exclude group: 'log4j'
```

Comment:
exclusion copied from other modules. 

Needed hadoop dependency for `Configuration` class in `RCKUtils`